packages/react/src/FhirPathTable/FhirPathTable.tsx

```diff
@@ -36,7 +36,7 @@ export interface SmartSearchResponse {
  */
 export function FhirPathTable(props: FhirPathTableProps): JSX.Element {
   const medplum = useMedplum();
-  const [schema, setSchema] = useState<IndexedStructureDefinition | undefined>();
+  const [schemaLoaded, setSchemaLoaded] = useState<boolean>(false);
```

Comment:
Is the generic type annotation needed here?  I would assume type inference should work correctly for this case